### PR TITLE
Update Readme Rolling

### PR DESCRIPTION
**Pull request by pablogs9**
This PR modifies the supported platform table in order to give more importance to Standalone build systems.

The idea is to slowly deprecate the micro_ros_setup package because it is proven that working with micro-ROS modules for specific platforms is easier to maintain.

I think that we cannot fully deprecate micro_ros_setup in Humble but IMO it is a good idea to start giving more importance to the micro-ROS components.

This changes this:

image

To this:

image

Thoughts and opinions @mamerlan @Acuadros95 @ralph-lange @JanStaschulat?

### CODE REVIEW

**Review by mamerlan — APPROVED** [review]
Parabens

**Comment by Acuadros95** [comment]
Maybe we should delete or modify the `Standalone build system tools` section, or at least modify this line:
> These other options are secondary tools and may not have full support for all features.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<br/>
+<a>
```
Comment (ralph-lange) [inline README.md:2]: Did you introduce the anchor element to link to micro.ros.org? Or shall it serve as anchor for navigation in the page?

**Review by Acuadros95 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pablogs9) [reply README.md]: Removed here 8e8be294e31b1cfa16adf3f710995965b7556a77

**Review by mamerlan — APPROVED** [review]

**Review by mamerlan — APPROVED** [review]

**Comment by mamerlan** [comment]
approved

**Comment by pablogs9** [comment]
@ralph-lange @JanStaschulat could you please confirm that you are ok with this?

**Review by JanStaschulat — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -18,46 +23,48 @@ This ROS 2 package is the entry point for building micro-ROS apps for different
...
-| Linux                                                                                                               | *Host <sup>2</sup>*                                                                                                                                                                      | Ubuntu 18.04/20.04           | `host`                       |
-| Android                                                                                                             | [AOSP](https://source.android.com) <sup>4</sup>                                                                                                                                          | Latest                       | `android generic`            |
+This package provides tools and utils to crosscompile micro-ROS with just the common ROS 2 tools for the following platforms platforms.
+
+**micro-ROS [components for each platform](#standalone-build-system-tools) are available and they are the preferred option.**
```
Comment (ralph-lange) [inline README.md:28]:
```suggestion
**Note that this package provides basic support only, intended in particular for ROS developers who are new to microcontrollers. Micro-ROS [components for each platform](#standalone-build-system-tools) are available and provide a deeper and more flexible integration with the platform-specific build systems.**
```

Similar to @Acuadros95's comment, I propose to also relax this formulation.

**Review by ralph-lange — APPROVED** [review]
LGTM. Thank you, @pablogs9.

**Comment by pablogs9** [comment]
CI is failing due to this: https://github.com/colcon/colcon-core/issues/503

So, I'm merging

**Pull request merged**